tests/Traits/KnplabsOrmTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\TranslationBundle\Tests\Model;
+
+use Doctrine\Common\EventManager;
+use Gedmo\Translatable\TranslatableListener;
+use Sonata\DoctrineORMAdminBundle\Datagrid\ProxyQuery;
+use Sonata\TranslationBundle\Enum\TranslationFilterMode;
+use Sonata\TranslationBundle\Filter\TranslationFieldFilter;
+use Sonata\TranslationBundle\Test\DoctrineOrmTestCase;
+use Sonata\TranslationBundle\Tests\Fixtures\Traits\ORM\Knplabs\Article;
+use Sonata\TranslationBundle\Tests\Fixtures\Traits\ORM\Knplabs\ArticleTranslation;
+
+/**
+ * @author Omar Jbara <omar.jbara2@gmail.com>
+ */
+class KnplabsOrmTest extends DoctrineOrmTestCase
+{
+    public const ARTICLE = Article::class;
+    public const TRANSLATION = ArticleTranslation::class;
+
+    /** @var TranslatableListener */
+    private $translatableListener;
+
+    protected function setUp()
```

Comment:
The tests will fail on Travis unless you provide a return type declaration.
```suggestion
    protected function setUp(): void
```